README.md

```diff
@@ -129,17 +129,55 @@ and run dump command:
 docker-compose run --rm db pg_dump -Fc -h db -v -U postgres feinstaub -f /opt/code/feinstaub-api-db.dump
 ```
 
---- 
+---
 
 ## Deployement
 
 To deploy the API, we use Dokku.
 
+### Postgres
+```
+sudo docker pull postgres:9.6
+
+sudo docker tag postgres:9.6 dokku/postgres:9.6
+
+dokku apps:create postgres
+
+sudo docker volume create --name feinstaub-db-data
+
+dokku docker-options:add postgres run,deploy --volume feinstaub-db-data:/var/lib/postgres
+
+dokku tags:deploy postgres 9.6
+
+# create database if non exists
+
+dokku enter postgres
+
+> createdb feinstaub -U postgres
 ```
```

Comment:
@chegejames Postgres would be optional because we'd expect RDS setup. Also let's document using [dokku-postgres plugin](https://github.com/dokku/dokku-postgres) instead. It's more straightforward.

Reply:
@DavidLemayian wanted to first deploy via dokku, their API as it is then make it robust in a subsequent PR. Will make these changes.

Reply:
@chegejames I don't understand your response. Could you expound? A dokku plugin is dokku deployment.. 🤔